src/app/(tabs)/scanner/index.tsx

```diff
@@ -60,13 +64,22 @@ export default function ScannerScreen() {
           style={styles.camera}
           type={CameraType.back}
           onMountError={handleCameraError}
-          ref={(ref) => camera = ref}
-          ratio="16:9"
+          ref={(ref) => (camera = ref)}
+          ratio="1:1"
+          barCodeScannerSettings={{
+            barCodeTypes: [BarCodeScanner.Constants.BarCodeType.qr],
+          }}
+          onBarCodeScanned={scanned ? undefined : handleBarCodeScanned}
         />
-        <Button onPress={takePicture} title="Scan" />
+        {scanned && (
+          <Button
+            title={"Tap to Scan Again"}
+            onPress={() => setScanned(false)}
+          />
+        )}
       </View>
     </Container>
-  )
+  );
 }
 
 const cameraWidth = 350 // used in a 4:3 ratio
```

Comment:
Since you changed the camera ratio, these constants will need to be changed as well. This is why the camera preview seems "distorted". You could simply set `const cameraRatio = 1;`